hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/7_8_0/6407-missing-conformity-to-ihe-balp.yaml

```
@@ -0,0 +1,5 @@
+---
+type: fix
+issue: 6407
+backport: 7.8.0
+title: "Corrected IHE BALP AuditEvent generation pr. resource owner. Thanks to Jens Villadsen (@jkiddo) for the contribution!"
```

question: what is a `generation pr. resource owner`? I don't understand this changelog. Maybe a better way to phrase this would be: 
```suggestion
title: "Corrected IHE BALP AuditEvent generation, so that it records one Audit Event per resource owner. Thanks to Jens Villadsen (@jkiddo) for the contribution!"
```

Is that an accurate description?

---

yeah - totally agree.